src/components/LogoutButton.react.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './css/logout.css';
+
+/**
+ * Logout button to submit a form post request to the `logout_url` prop.
+ */
+export default class LogoutButton extends React.Component {
+    render() {
+        const {id, logout_url, label, className, style} = this.props;
+        return (
+            <form
+                action={logout_url}
+                method="post"
+                className="dash-logout-frame"
+            >
+                <button
+                    className={`dash-logout-btn ${className}`}
```

Comment:
Here if `className` is undefined, it will add "undefined" to the class. Would be cleaner to check if `className` exists before appending it.